localstack/config.py

```diff
@@ -735,100 +737,128 @@ def get_gateway_listen(gateway_listen: str) -> List[HostAndPort]:
 # Disable the check for MaxNumberOfMessage in SQS ReceiveMessage
 SQS_DISABLE_MAX_NUMBER_OF_MESSAGE_LIMIT = is_env_true("SQS_DISABLE_MAX_NUMBER_OF_MESSAGE_LIMIT")
 
-# host under which the LocalStack services are available from Lambda Docker containers
+# DEPRECATED: only applies to old lambda provider
+# Endpoint host under which LocalStack APIs are accessible from Lambda Docker containers.
 HOSTNAME_FROM_LAMBDA = os.environ.get("HOSTNAME_FROM_LAMBDA", "").strip()
```

Comment:
not sure if we're not actually gonna have to keep that for some edge cases :thinking:

Reply:
agreed. We might want adopt the new `host` terminology instead of `hostname` in that case 🤔